Sources/Danger/GitLabDSL.swift

```diff
@@ -230,7 +230,7 @@ extension GitLab {
         }
 
         public let avatarUrl: String?
-        public let id: Int // swiftlint:disable:this identifier_name
+        public let id: Int
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitLabDSL.swift;
  Line: 233;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)